openedx/stanford/pavelib/utils/i18n_helpers.py

```diff
@@ -151,32 +290,73 @@ def fix_metadata(pofile):
     pofile.metadata.update(fixes)
 
 
-def merge_existing_translations(existing, target, lang):
-    """
-    Merge translations from existing file to target file for locale lang and
-    push up to Transifex
-    """
-    from openedx.stanford.pavelib.i18n import CONFIG
-    target_filename = "stanford_" + target + '.po'
+def merge_translations():
+    for language in CONFIG.translated_locales:
+        LOG.info("Merging language: %s...", language)
+        if not CONFIG.get_messages_dir(language).exists():
+            # Language not yet available in code, fetch from upstream Transifex
+            LOG.warn("Fetch upstream translations manually for %s", language)
+            break
+        merge_mappings = {
+            'django.po': ['django', 'mako'],
+            'djangojs.po': ['djangojs', 'underscore'],
+        }
+        for existing, targets in merge_mappings.items():
+            for target in targets:
+                _merge_existing_translations(existing, target, language)
+        LOG.info("Merged language: %s.", language)
 
-    # Fetch common messages first
-    msgcomm_template = 'msgcomm {existing_file} {target_source} -o {output}'
-    target_source = CONFIG.source_messages_dir / target_filename
-    common_file = CONFIG.get_messages_dir(lang) / 'common.po'
-    msgcomm_cmd = msgcomm_template.format(
-        existing_file=CONFIG.get_messages_dir(lang) / existing,
+
+def _merge_common_fetch(target_source, target_filename, common_file, language, existing_file):
```

Comment:
`target_filename` is not needed, `existing_file` should be `existing`

Reply:
Done.